SPEC.md

```diff
@@ -215,6 +215,7 @@ The following environment variables MUST be set for each application's main proc
 * **AC_METADATA_URL** URL where the [metadata service](#app-container-metadata-service) for this pod can be found.
 
 An executor MAY set additional environment variables for the application processes.
+`$PATH` MAY contain more directories than the ones listed.
```

Comment:
Hi Jonathan, 

sorry for delayed answer - second baby arrived last week ;-)

Regarding $PATH: Maybe I misunderstood sth but I thought it makes a lot of sense to be able to optionally allow additional directories already in the execution environment.
Why?
For example when an image contains different versions of an executable in different directory trees (say, /.../stable/myapp vs. /.../devel/myapp, from, say repo checkouts) - then you could in a PaaS like surrounding framework allow the user to simply switch the version of what is executed, globally and simple. Sure you could invent indirections via other environment variables, checked by wrapper scripts you could add inside - but I find $PATH to be the most direct and most transparent way for user to control what is started in such a setup.

Maybe I misunderstand the philosophy of appc and you want to get as close to freshly "booted" VMs as possible but since you allow explicitly "additional environment variables" in the principal exec env I would say the $PATH should also be allowed to be widened.